### DISCUSSION TITLE
Not all results shown

### DISCUSSION THREAD

**Discussion by Simplifier** [community/community#122007] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Code search doesn't work properly. For example, query https://github.com/search?q=flask+language%3APython&type=code shows only 5 pages of results, despite a badge near the Code tab shows that 1.9M files are found

![Monosnap Code search results - Google Chrome 2024-](https://github.com/community/community/assets/3374612/ac4ac3e8-13f4-4e70-a8c0-3ed0876eaebb)

**Comment by look** [comment, marked as answer]
Thanks for your feedback. It is a current limitation of GitHub code search that it only shows 100 results. As a workaround, please make your search more selective.

**Reply by Simplifier** [reply]
Wow, this is pretty sad =(
